DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
```

Comment:
there seems to be some legal concerns about not having them, which is
why we did them in the cmf and in FOS libraries. but i agree that they
seem pretty pointless, will remove them happily :-)